tools/cbmc/proofs/ARP/ARP_OutputARPRequest_buffer_alloc2/config_minimal_configuration_minimal_packet_size/Makefile.json

```diff
@@ -16,6 +16,7 @@
     "ipconfigNUM_NETWORK_BUFFER_DESCRIPTORS=2",
     "configUSE_TRACE_FACILITY=0",
     "configGENERATE_RUN_TIME_STATS=0",
-    "ipconfigETHERNET_MINIMUM_PACKET_BYTES={MINIMUM_PACKET_BYTES}"
+    "ipconfigETHERNET_MINIMUM_PACKET_BYTES={MINIMUM_PACKET_BYTES}",
+    "ipconfigUSE_TCP=1"
```

Comment:
This change is unrelated to this commit. (But anyway, this entire file should be removed I think)